src/librustc_resolve/macros.rs

```diff
@@ -172,7 +176,8 @@ impl<'a> base::Resolver for Resolver<'a> {
         ImportResolver { resolver: self }.resolve_imports()
     }
 
-    fn find_attr_invoc(&mut self, attrs: &mut Vec<ast::Attribute>) -> Option<ast::Attribute> {
+    fn find_legacy_attr_invoc(&mut self, attrs: &mut Vec<ast::Attribute>)
```

Comment:
It only resolves legacy attribute macro invocations and legacy derive invocations (i.e. attributes and derives from `#![plugin(...)]`).